extensions/amp-story-subscriptions/0.1/amp-story-subscriptions.js

```diff
@@ -62,6 +69,27 @@ export class AmpStorySubscriptions extends AMP.BaseElement {
       this.subscriptionService_ = subscriptionService;
       this.localizationService_ = localizationService;
 
+      const pages = this.win.document.querySelectorAll(
+        'amp-story-page:not([ad])'
+      );
+      devAssert(
+        pages.length >= 4,
+        'The number of pages should be at least 4 to enable subscriptions feature, got %s',
+        pages.length
+      );
+
+      const subscriptionsPageIndex = this.element.getAttribute(
+        'subscriptions-page-index'
+      );
+      this.storeService_.dispatch(
+        Action.SET_SUBSCRIPTIONS_PAGE_INDEX,
+        clamp(
+          subscriptionsPageIndex,
```

Comment:
Does this work without `parseInt`?

Reply:
Nice catch! Added the parseInt and updated the example to make sure it works.